src/index.html

```diff
@@ -15,8 +15,56 @@
       rel="stylesheet"
       href="./style.css"
     />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#imagem">
+        <img
+          src="images/logo.png"
+          alt="moyo"
+          class="image"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul>
+          <li>
+            <a
+              href="#item1"
+              class="is-active"
+            >
+              Iphone
+            </a>
+          </li>
+
+          <li><a href="#item2">samsung</a></li>
+          <li><a href="#item3">smartphone</a></li>
+          <li>
+            <a
+              href="#item4"
+              data-qa="hover"
+              class="laptops"
```

Comment:
Class `laptops` ? This shouldn't be here.
All links should have a single class called `nav__link`